_1327/main/views.py

```diff
@@ -8,6 +10,9 @@
 from _1327.documents.models import Document
 from _1327.documents.utils import permission_warning
 
+from .forms import MenuItemForm
```

Comment:
do we want relative imports here? Normally we are using absolute imports... we can change this, but then we should change it everywhere